.nvmrc

```diff
@@ -0,0 +1 @@
+16.14.0
```

Comment:
Why do we need this btw?

Reply:
So you can use the recommended version of node with "nvm use".

I like to have always the versions of the npm packages and node version declared. If some people want to build this the steps are easy. 

"nvm use" and  "npm install"

Reply:
I see.
But what does it do?
Or what does it help to prevent?

Reply:
> I see. But what does it do? Or what does it help to prevent?

For example if you work with an old node version or newer in some cases, some libraries in our package.json will not work. And will show error because the incompatibility. 

This happen to me in so many projects... So I like to always declare all the version in package.json and the node version for this to be use.

Reply:
Alright. Not a problem I really faced before but that fine 👍🏼

Reply:
> Alright. Not a problem I really faced before but that fine 👍🏼

Its not usual, but i'had some old projects (4-6 years old) and this happens oftern. (Big projects with ton of dependecies). Maybe in this case its overkill. But just in case.